recipes/magic_enum/all/conanfile.py

```diff
@@ -60,7 +60,7 @@ def build(self):
         pass
 
     def package(self):
-        copy(self, "*", src=os.path.join(self.source_folder, "include"), dst=os.path.join(self.package_folder, "include"))
+        copy(self, "*", src=os.path.join(self.source_folder, "include/magic_enum"), dst=os.path.join(self.package_folder, "include"))
```

Comment:
```suggestion
        if Version(self.version) < "0.9.4":
            copy(self, "*", src=os.path.join(self.source_folder, "include"), dst=os.path.join(self.package_folder, "include"))
        else:
            copy(self, "*", src=os.path.join(self.source_folder, "include", "magic_enum"), dst=os.path.join(self.package_folder, "include"))
```

Reply:
@SpaceIm Thanks for the feedback regarding support of all package versions of `magic_enum`.

In order to achieve that I've changed the `package()` method to use the CMake install target.